examples/xgboost/histogram-based/README.md

```diff
@@ -0,0 +1,21 @@
+# Histogram-based Federated Learning for XGBoost   
+
+## Run automated experiments
+To run this example with NVFlare, follow the below steps.
+
+### Environment Preparation
+
+1. git clone [xgboost](https://github.com/dmlc/xgboost)
+2. build and install xgboost-federated following [README](https://github.com/dmlc/xgboost/blob/master/plugin/federated/README.md)
+
+### Run local experiments with simulator
+Next, we will use the NVFlare simulator to run FL training automatically.
+```
+bash run_experiment_simulator.sh
+```
+
+## Results on 5- and 20-client under various training settings
```

Comment:
Result on centralized training (without using parallel trees) - w/o part may not be needed

Reply:
got it, I leave that to you for a follow up PR. thanks